src/styles/card.css

```diff
@@ -0,0 +1,96 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: "Roboto", sans-serif;
+  color: var(--color-black);
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.product-card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card__name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+
+  margin-bottom: 4px;
+}
+
+.product-card__code {
+  font-size: 10px;
+  color: var(--color-gray);
+
+  margin-bottom: 16px;
```

Comment:
3. [BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under the element selector.
   It would be best if you use 2 classes, one for positioning and the second for styles.

Reply:
She didn't set margins to a block, just for elements of this block, from my perspective it's valid

Reply:
thanks, margins for elements of the block are allowed indeed :)